Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs

```diff
@@ -882,8 +892,14 @@ bool proposal
                     BarCodeText = thisClass.AttendanceMode != AttendanceMode.Online ? Faker.Random.String(length: 10, minChar: '0', maxChar: '9') : null,
                     MeetingUrl = thisClass.AttendanceMode != AttendanceMode.Offline ? new Uri(Faker.Internet.Url()) : null,
                     MeetingId = thisClass.AttendanceMode != AttendanceMode.Offline ? Faker.Random.String(length: 10, minChar: '0', maxChar: '9') : null,
-                    MeetingPassword = thisClass.AttendanceMode != AttendanceMode.Offline ? Faker.Random.String(length: 10, minChar: '0', maxChar: '9') : null
+                    MeetingPassword = thisClass.AttendanceMode != AttendanceMode.Offline ? Faker.Random.String(length: 10, minChar: '0', maxChar: '9') : null,
+                    AttendeeEmail = attendees.Count > 0 ? attendees[i].Email : null,
```

Comment:
Does this check make sense? Could we be out-of-bounds of the array if `i` was 2 and the attendee count was 1 (due to the `Where` clause flagged earlier).

Perhaps something more defensive such as:

```suggestion
                    AttendeeEmail = attendees.Count > i ? attendees[i].Email : null,
``` 

Also note any amends to this pattern need to be replicated below also

Reply:
I'm going to change this to be a serialised string which will remove all of these issues